src/orchestrator/Orchestrator.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import * as lunatic from "@inseefr/lunatic";
+import * as lunaticEDT from "lunatic-edt";
+import { Button } from "@mui/material";
+
+const { ThemeProvider, ...edtComponents } = lunaticEDT;
+
+export type Props = {
+    goPrevious: () => void;
+    goNext: () => void;
+    isLast: boolean;
+    isFirst: boolean;
+};
+const Pager = (props: Props) => {
+    const { goPrevious, goNext, isLast, isFirst } = props;
+
+    return (
+        <div className="pagination">
+            <Button onClick={goPrevious} disabled={isFirst}>
+                Previous
+            </Button>
+            <Button onClick={goNext} disabled={isLast}>
+                Next
+            </Button>
+        </div>
+    );
+};
+
+const onLogChange = (e: React.ChangeEvent<HTMLInputElement>) => console.log("onChange", { ...e });
+export type OrchestratorProps = {
+    source: object;
+    data?: object;
+};
+export const OrchestratorForStories = (props: OrchestratorProps) => {
+    const { source, data } = props;
+    const { goPreviousPage, goNextPage, isLastPage, isFirstPage, getComponents, getCurrentErrors } =
+        lunatic.useLunatic(source, data, {
+            onChange: onLogChange,
+        });
+    const components = getComponents();
+    const currentErrors = getCurrentErrors();
+
+    return (
+        <ThemeProvider>
```

Comment:
Orchestrator is Already Scoped into <ThemeProvider>

Reply:
This on I don't understand

Reply:
if you look [here](https://github.com/InseeFrLab/edt/blob/feat/offline/src/App.tsx#L25-L29) you see that Orchestrator is inside <ThemeProvider> so should not rewrap by <ThemeProvider> in this file